scripts/prepare.sh

```diff
@@ -47,6 +47,7 @@ else
 echo "Linking the specified local repo from ${LOCAL_NEARUP_SRC} to ${NEARUP_SRC}"
 ln -s $LOCAL_NEARUP_SRC $NEARUP_SRC
 fi
+cd $NEARUP_SRC && git checkout 0.1.2
```

Comment:
1. It seems that by doing `cd` here you break the next commands
2. Why don't we just git clone the fixed tag in the first place?
3. It does not seem sane to force the local nearup_src to some tagged version instead of whatever development version it is there, does it?

Reply:
Yes, thats true will change that I was just about to do that :100:

Reply:
Ah, the problem is only that you have to checkout the commit on the LOCAL_NEARUP_SRC if somebody is using that so can't just clone. Reversed the order though now.

Reply:
The master after that commit is not compatible to new nearup interface so we have to force it, until @mhalambek creates a PR to download the pypi package and migrates the commands to the new interface instead of using a checkout of the repo.